Formula/leafnode.rb

```diff
@@ -14,6 +14,89 @@ def install
                           "--with-user=#{ENV["USER"]}", "--with-group=admin",
                           "--sysconfdir=#{etc}/leafnode", "--with-spooldir=#{var}/spool/news/leafnode"
     system "make", "install"
+    (prefix/"homebrew.mxcl.fetchnews.plist").write fetchnews_plist
+    (prefix/"homebrew.mxcl.texpire.plist").write texpire_plist
+  end
+
+  def caveats; <<-EOS.undent
```

Comment:
I thought we produced a generic message where multiple `plists` exists? i.e. tell people to symlink with `*` rather than a direct path? Is that not the case? Maybe I'm misremembering 😅.